second_task/go/src/square/square.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	//	"math"
+)
+
+type Point struct {
+	x, y int
+}
+
+type Square struct {
+	start Point
+	a     uint
+}
+
+func (my_square Square) End() (x_result int, y_result int) {
```

Comment:
For functions receivers short names are preferable

Reply:
I don't understand meaning of this comment. Can u explain it?

Reply:
In this case my_square is function receiver name, you use too long name for function receiver name as a common practice function receiver equal first letter or some reduced word like server -> srv, request -> req, Square -> s

Reply:
understood.